leaflet-beautify-marker-icon.js

```diff
@@ -123,6 +123,7 @@
                 // having a marker requires an extra parent div for rotation correction
                 if (this.options.iconShape === 'marker') { 
                     var wrapperDiv = document.createElement('div');
+                    wrapperDiv.className = 'beautify-marker-wrapper';
```

Comment:
what if we just change the className to 'beautify-marker' instead of 'beautify-marker-wrapper', then we don't need to change the css file.

Reply:
It looks like working fine.I've just created a separate class to isolate future changes